plusplusbot/scorekeeper.py

```diff
@@ -125,7 +122,10 @@ def leaderboard(self, channel, limit=leaderboard_limit):
         return sorted(self.scoreboard[channel]["scores"].items(), key=lambda i: (i[1], i[0]), reverse=True)[:limit]
 
     def history(self, channel, limit=history_limit):
-        return self.scoreboard[channel]["history"][::-1][:limit]
+        return self.raw_history(channel)[-limit:][::-1]
```

Comment:
Is there a different in flipping around the logic here?

The reversing is done after the limit, which actually makes more sense!
But the limit is like a negative start now?

Reply:
Reversal is, well, not exactly expensive but small thing that can go a long way. Going through the whole list once is usually avoided unless you absolutely have to.. So minimising the number of items to reverse was my natural instinct. Happy to revert it if you don't like it. The data is too small to make a difference.